### PR TITLE
Hdds 3772. Add LOG to S3ErrorTable for easier problem locating. 

### PR DESCRIPTION
**Pull request by ChenSammi**
https://issues.apache.org/jira/browse/HDDS-3772

### CODE REVIEW

File: hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java [hunk 1]
```diff
@@ -29,6 +31,9 @@
  */
 public final class S3ErrorTable {
 
+  private final static Logger LOG = LoggerFactory.getLogger(
+      S3ErrorTable.class);
```
Comment (xiaoyuyao) [inline hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java:35]: This seems to be the main part matching scope of the title. Should we use separate tickets to track other changes?

Comment (ChenSammi) [reply hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java:35]: Sorry, Xiaoyu, I will upload a clean patch.

File: hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java [hunk 2]
```diff
@@ -106,6 +111,7 @@ public static OS3Exception newError(OS3Exception e, String resource) {
     OS3Exception err =  new OS3Exception(e.getCode(), e.getErrorMessage(),
         e.getHttpCode());
     err.setResource(resource);
+    LOG.info(err.toXml());
```
Comment (bharatviswa504) [inline hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java:114]: One comment:
This should be LOG.error

**Comment by bharatviswa504** [comment]
Thank You @ChenSammi for the contribution and @xiaoyuyao for the review.

**Pull request merged**